packages/flutter_localizations/test/material/translations_test.dart

```diff
@@ -607,4 +607,15 @@ void main() {
     expect(localizations, isA<MaterialLocalizationIt>());
     expect(localizations.dateHelpText, 'gg/mm/aaaa');
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/176677
+  testWidgets('Basque translation for timeOfDayFormat', (WidgetTester tester) async {
+    const Locale locale = Locale('eu');
+    expect(GlobalCupertinoLocalizations.delegate.isSupported(locale), isTrue);
```

Comment:
Oh I think some of the tests above might just copy paste tests from cupertino localizations and we didn't notice it! 😂

```suggestion
    expect(GlobalMaterialLocalizations.delegate.isSupported(locale), isTrue);
```

Reply:
Good catch! I'll go ahead and fix the other ones too.